### PR TITLE
Update the README to include some notes on how to build locally.

### PR DESCRIPTION
**Pull request by mblayman**
The config also used a deprecated option which I updated.

### CODE REVIEW

**Comment by kinow** [comment]
+1 :+1:

**Comment by mblayman** [comment]
@kinow, is there some action I'm supposed to take here? I don't have write access to the repo so I can't merge this branch myself.

**Comment by kinow** [comment]
Hi @mblayman , I have access to merge it, but @isaacs, @jonathanKingston  and others have been more active on the project, so I prefer to give them time to review it first :-)

**Comment by mblayman** [comment]
Cool, thanks. I wanted to make sure I wasn't missing something obvious. :smile:

**Comment by kinow** [comment]
Not really, thanks for the contributions :-) feel free to bump the issue if it takes too long to get any reply.

**Comment by mblayman** [comment]
Hi, guys. I'd love to join the TestAnything organization and get write access to the repository so I can make some solid contributions here in a timely manner. I know I don't have a very long history of contributions to these particular repositories, but I am the primary author and maintainer of tappy and I'm heavily invested into making the TAP community better.

I'd like to see testanything.org be a great marketing tool for the protocol, and I'd really like to help clean up the site. Getting write access to the repository could help me make these contributions without being a big drag on your time. I would still seek out a :+1: from others, but I could free you from the actual merging and maintenance work. Thanks!

**Comment by mblayman** [comment]
@kinow, this PR has been sitting around for a couple of weeks now without any additional attention. Perhaps Jonathan and/or Isaac are busy with other projects. Could you merge this work or (preferably) give me write permission to the repo so I could merge it myself? Thanks in advance!

**Comment by mblayman** [comment]
Hi, again! This branch remains unmerged for three weeks now. Could someone merge this or give me permission to merge it myself? Thanks!

**Comment by jonathanKingston** [comment]
Added one comment, happy to merge after that explanation.

File: _config.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 name: TestAnything.org
 markdown: redcarpet
-pygments: true
+highlighter: pygments
```
Comment (mblayman) [inline _config.yml]: It's a syntax change in jekyll. I only know this because I got a deprecation warning when I ran `jekyll serve`.

**Pull request merged**